packages/desktop-client/src/components/reports/ModeButton.tsx

```diff
@@ -8,20 +8,22 @@ type ModeButtonProps = {
   children: ReactNode;
   style?: CSSProperties;
   onSelect: MouseEventHandler<HTMLButtonElement>;
+  color?: string;
 };
 
 export function ModeButton({
   selected,
   children,
   style,
   onSelect,
+  color = theme.menuBackground,
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏instead of adding a prop for each style attribute - it would be better to instead override the bg color via using the existing `style` prop.

Reply:
Yea, I started with that but it then overrides the hover and select styles that are being used in this function. So option in Spending.tsx is what I have now:


> 
            color="inherit"

or:

> 
            style={{
              backgroundColor: 'inherit',
              ...(['lastMonth', 'twoMonthsPrevious'].includes(mode) && {
                backgroundColor: theme.buttonPrimaryBackground,
                ':hover': {
                  backgroundColor: theme.buttonPrimaryBackgroundHover,
                },
              }),
            }}

I don't mind either way. Which do you prefer? Or maybe you have an option C?

Reply:
What about adding a `selectedStyles` prop and then moving `styles` up to line ~28? That would allow overriding both parts individually.

Reply:
Yea, good shout. Actually not using the "style" prop in any of the "modeButton" uses. So I'll just move that up above the selected/hover attributes and Bob's your uncle. Cheers!